lib/qx/tool/cli/commands/Compile.js

```diff
@@ -70,6 +66,11 @@ qx.Class.define("qx.tool.cli.commands.Compile", {
             type: "string",
             array: true
           },
+          "update-po-files": {
+            describe: "enables detection of translations and writing them out into .po files",
```

Comment:
Maybe provide an alias such as `-u` since this is a long option name?

Reply:
no problemo